js/coinone.js

```diff
@@ -371,78 +371,253 @@ module.exports = class coinone extends Exchange {
-                market = this.markets_by_id[marketId];
+        let status = undefined;
+        const orderStatus = this.safeString (order, 'status');
+        if (orderStatus === 'live') {
+            status = 'open';
+            filled = 0;
+            if (amount > remaining) {
+                amount = remaining;
+                remaining = 0;
             }
+        } else if (orderStatus === 'partially_filled') {
+            status = 'open';
+            filled = amount - remaining;
+        } else {    // 'filled'
+            status = 'closed';
+            filled = amount - remaining;
         }
-        if (market !== undefined) {
-            symbol = market['symbol'];
-        }
+        const cost = price * filled;
         return {
-            'info': order,
             'id': id,
             'clientOrderId': undefined,
-            'timestamp': timestamp,
             'datetime': this.iso8601 (timestamp),
+            'timestamp': timestamp,
             'lastTradeTimestamp': undefined,
+            'status': status,
             'symbol': symbol,
             'type': 'limit',
             'side': side,
             'price': price,
-            'cost': cost,
             'amount': amount,
             'filled': filled,
             'remaining': remaining,
-            'status': status,
-            'fee': fee,
-            'average': undefined,
+            'cost': cost,
             'trades': undefined,
+            'fee': {
+                'currency': feeCurrency,
+                'cost': this.safeFloat (info, 'fee'),
+                'rate': this.safeFloat (info, 'feeRate'),
+            },
+            'info': order,
         };
     }
 
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        // [WARNING] Because of the limited capabilities of Coinone V2 API, you should be noticed below
+        // * The symbol parameter must be given. If not, an exception will occur.
+        // * Always returns the list of the all open orders with the symbol: The since and limit parameters are totally ignored.
+        // * The returned amount might not be same as the ordered amount. If an order is partially filled, the returned amount means the remaining amount.
+        // * For the same reason, the returned amount and remaining are always same, and the returned filled and cost are always zero.
+        await this.loadMarkets ();
+        if (symbol === undefined) {
+            throw new ExchangeError (this.id + ' allows fetching closed orders with a specific symbol');
+        }
+        const market = this.market (symbol);
+        const request = {
+            'currency': market['id'],
+        };
+        const response = await this.privatePostOrderLimitOrders (this.extend (request, params));
+        const result = [];
+        const limitOrders = this.safeValue (response, 'limitOrders');
+        for (let i = 0; i < limitOrders.length; i++) {
```

Comment:
We need to move this parsing section to `parseOrder()`, to keep it in line with almost all other exchanges.

Reply:
same above